models/issue_comment.go

```diff
@@ -110,6 +110,45 @@ const (
 	CommentTypeChangeIssueRef
 )
 
+func (t CommentType) String() string {
+	return []string{
```

Comment:
If this will be optimized by compiler? Or we may need to store the slice as global variable.

Reply:
I don't understand you, and it worked for me. Could you explain this or should I just add it as a global var?

Reply:
Yeah, I mean maybe put that slice as a global var may have better performance although it's slight. I don't make sure Golang compiler could do that itself.